docs/reference/core_concepts/expectations/standard_arguments.rst

```diff
@@ -57,7 +57,7 @@ All Expectations accept a boolean `catch_exceptions` parameter. If true, executi
 
     {
         "result": False,
-        "raised_exception": True,
+        "catch_exceptions": True,
```

Comment:
Is this correct? I think the *return* value *is* `raised_exception`

Reply:
I believe catch_exception worked in my notebook and raised_exception did not, it is also under the catch_exceptions heading

Reply:
Got it. I think the issue is that the sentence above is confusing: what's shown in the code block is the *return* value , which *is* "raised_exception". Probably the best fix is to change the sentence above to be better; something like:

All Expectations accept a boolean ``catch_exceptions`` parameter. If that parameter is set to ``True``, then Great Expectations will intercept any exceptions so that execution will not fail if the Expectation encounters an error. If Great Excpectations catches an exception while evaluating an Expectation, the Expectation result will include  ``success: False``, ``raised_exception: True``, and (in `BASIC` and `SUMMARY` modes) the traceback under the ``exception_traceback`` key.

Reply:
Clarified it in the last commit! I believe it should be clear now!

Reply:
^ Just did this!